_includes/tutorials/ksql-concatenation-columns/ksql/markup/dev/transient-reporting.adoc

```diff
@@ -0,0 +1,35 @@
+Now we write a query where we select the desired fields but we don't have them listed in a comma separated list.  Instead we use the `+` operator between the fields causting them to get concatenated together.
```

Comment:
```suggestion
Now we write a query where we select the desired fields but we don't have them listed in a comma separated list.  Instead we use the `+` operator between the fields causing them to get concatenated together.
```
^ If you don't move forward with my other suggestion, I think you meant 'causing'